### ISSUE TITLE
Remove api 1.x usage from infrastructure/docker scripts

### ISSUE THREAD

**Issue by mitchell852** [apache/trafficcontrol#6039]
## I'm submitting a ...
-  improvement request (usability, performance, tech debt, etc.)

## Traffic Control components affected ...

-  other

## Current behavior:
There are also several instances of `api/1.x` being used in:
`infrastructure/docker/traffic_server_edge/run.sh`
`infrastructure/docker/traffic_server_mid/run.sh`
`infrastructure/docker/traffic_monitor/run.sh`
`infrastructure/docker/README.md`
`infrastructure/docker/traffic_ops/run.sh`
`infrastructure/docker/traffic_stats/run.sh`
`infrastructure/docker/traffic_vault/run.sh`
`infrastructure/docker/traffic_router/run.sh`

## New behavior:
1.x will be removed from TC 6.0. Failure to migrate to api 2.x+ will result in broken scripts.

## Minimal reproduction of the problem with instructions:
N/A

**Comment by mitchell852** [comment]
marked as high impact as i'm guessing these scripts are broken atm.

**Comment by ocket8888** [comment]
Yeah but I've never heard of anyone besides @rob05c using them, and he hasn't complained for at least 5 months. So the impact of this is actually nearly zero.

**Comment by mitchell852** [comment]
well, if you use it the impact to you is high. :)

**Comment by ocket8888** [comment]
frankly I consider those experimental. No part of ATC depends on them, they aren't documented or (clearly) supported.

**Comment by rob05c** [comment]
I would sadly agree with @ocket8888 - I'm not aware of anyone else using them either. 

I don't rebuild from scratch often, but I do use the containers from them frequently.

I've been intending to fix this, just haven't been able to find time.
